### DISCUSSION TITLE
How can u use templates in fastapi?

### DISCUSSION THREAD

**Discussion by cosmos369** [fastapi/fastapi#2536] [Questions] [answered]
I am planning to switch to fastapi now. But my only fear is the small community. I am new to programming. So, help and support is a priority for me which I get in Django/flask. But since fastapi is so clean and fast, my heart is going in that direction.

**Comment by Mause** [comment, marked as answer]
We have documentation on the use of templates: https://fastapi.tiangolo.com/advanced/templates/

**Comment by Kludex** [comment]
@cosmos369 Welcome! :tada: 

Our community members are very active on GitHub issues and other communication channels. :sunglasses: :+1:
